generate-spec

```diff
@@ -412,217 +433,223 @@ foreach ($parsedRoutes as $key => $value) {
 
 $tagNames = [];
 if ($useTags) {
-	foreach ($routes as $route) {
-		if (!in_array($route->tag, $tagNames)) {
-			$tagNames[] = $route->tag;
+	foreach ($routes as $scope => $scopeRoutes) {
+		foreach ($scopeRoutes as $route) {
+			if (!in_array($route->tag, $tagNames)) {
+				$tagNames[] = $route->tag;
+			}
 		}
 	}
 }
 
-foreach ($routes as $route) {
```

Comment:
Just had to retab, so better review with https://github.com/nextcloud/openapi-extractor/pull/24/files?w=1